scripts/generate_env.py

```diff
@@ -0,0 +1,53 @@
+import os
```

Comment:
it is better to show some log information to demonstrate the process in the terminal.